website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -160,16 +188,129 @@ yq e '.rds.secretName = env(RDS_SECRET)' -i charts/common/aws-secrets-manager/rd
             {{< /tab >}}
             {{< /tabpane >}} 
 
+### 2.4 Manual S3 Setup
+   1. Export values:
+      ```bash
+      export S3_SECRET="<your s3 secret name>"
+      export S3_BUCKET="<your s3 bucket name>"
+      export MINIO_SERVICE_HOST=s3.amazonaws.com
+      ```
+
+   1. [IRSA ONLY] IRSA Roles
+
+      1. Associate an OpenID Connect (OIDC) provider with your IAM role to authenticate your cluster with the IAM service.
```

Comment:
An OIDC provider must exist for your cluster to use IRSA. Create an OIDC provider and associate it with your EKS cluster by running the following command if your cluster doesn’t already have one: